.gitignore

```diff
@@ -2,6 +2,7 @@
 
 # Packages
 *.egg
+*.eggs
```

Comment:
Minor curiosity: I know what egg is but I have never seen a file/directory with ```.eggs``` extension. What are they for? Do you know which tool creates them?